manifest.json

```diff
@@ -4,6 +4,11 @@
     "short_name": "DGtW",
     "description": "'Degender the Web' replaces gendered pronouns on most Web pages with 'they/them/their'.",
     "version": "0.8.0",
```

Comment:
Please bump the version number to 0.9.0. This should also be done in package.json and package-lock.json.